### ISSUE TITLE
Text region detection with yolo v3, will it work?

### ISSUE THREAD

**Issue by KYZQ** [pjreddie/darknet#957]
It's just my idea，I konw yolo v3 is designed for objection detection. If I use it to detect text region and figure region(Two classification problems), will I success? 
The link is my same question with image in stack, but no one answer me .....
[https://stackoverflow.com/questions/51239481/text-region-detection-with-yolo-v3](url)
 Could any one give me some hints? Please!

**Comment by JohnWnx** [comment]
Yeah i’m interested to know how I could use it for text detection too

**Comment by liben2018** [comment]
you can re-calculate your anchor, then have a try.

**Comment by dhkim0225** [comment]
@KYZQ Did you try it? how was the result?

**Comment by pysongh** [comment]
I found a related answers from :
https://groups.google.com/forum/m/#!topic/darknet/VMJKa3Q6r1c

**Comment by priyamehta01** [comment]
@liben2018 : Hey, I am using YOLOv3 for character detection. Do you have any possible suggestions to recalculate the anchors?

**Comment by liben2018** [comment]
@priyamehta01, please find out the suggestions in [https://github.com/AlexeyAB/darknet](url)

**Comment by RahulRajote** [comment]
Is it possible to ready image data using yolov3

**Comment by theSage21** [comment]
@RahulRajote I am not sure what you mean. yolo is an image processing system. so it is in fact not only possible but the basis on which yolo works.
